pdal/plang/Invocation.cpp

```diff
@@ -268,22 +265,145 @@ bool Invocation::execute()
     Py_INCREF(m_varsOut);
     Py_ssize_t numArgs = argCount(m_function);
     m_scriptArgs = PyTuple_New(numArgs);
+
     PyTuple_SetItem(m_scriptArgs, 0, m_varsIn);
     if (numArgs > 1)
         PyTuple_SetItem(m_scriptArgs, 1, m_varsOut);
-    if (numArgs > 2)
-        PyTuple_SetItem(m_scriptArgs, 2, m_metaIn);
-    if (numArgs > 3)
-        PyTuple_SetItem(m_scriptArgs, 3, m_metaOut);
 
-    m_scriptResult = PyObject_CallObject(m_function, m_scriptArgs);
+
+    int success = PyModule_AddObject(m_module, "metadata", m_metadata_PyObject);
```

Comment:
These strike me as kind of pointless error handling.  A user can't do anything with the info and a developer is going to get a crash to examine pretty quickly.  If this kind of thing fails, something is really wrong that has nothing to do with metadata.